docs/data/quick-start.md

```diff
@@ -0,0 +1,791 @@
+    ```
+
+## 2. Send data
+
+=== "Browser"
+
+    ```ts title="TypeScript"
+    import { track } from '@amplitude/analytics-browser';
+
+    const eventProperties = {
+      buttonColor: 'primary',
+    };
+    track('Button Clicked', eventProperties);
+
+    ```
+
+=== "Node"
+
+    ```ts title="TypeScript"
+
+    import { track } from '@amplitude/analytics-node';
+
+    const eventProperties = {
+      buttonColor: 'primary',
+    };
+
+    track('Button Clicked', eventProperties, {
+      user_id: 'user@amplitude.com',
+    });
+    ```
+
+=== "Android"
+
+    ```kotlin title="Kotlin"
+
+    amplitude.track("eventType", mutableMapOf<String, Any?>("test" to "event property value"))
+    ```
+
+    ```java title="Java"
+    amplitude.trak("eventType", new HashMap() {{
+        put("test", "test event property value");
+    }});
+    ```
+
+=== "iOS"
+
+    ```obj-c title="Objective-C"
+    [[Amplitude instance] logEvent:@"Button Clicked"];
+    ```
+
+    ```swift title="Swift"
+    Amplitude.instance().logEvent("Button Click")
+    ```
+
+=== "Java"
+
+    ```java
+
+    amplitude.logEvent(new Event("Button Clicked", "test_user_id"));
+    ```
+
+=== "Python"
+
+    ```python title="Python"
+    from amplitude import BaseEvent
+
+    amplitude.track(
+        BaseEvent(
+            event_type="type of event",
+            user_id="USER_ID",
+            device_id="DEVICE_ID",
+            event_properties={
+                "source": "notification"
+            }
+    ))
+    ```
+
+=== "React Native"
+
+    ```ts title="TypeScript"
+
+    import { track } from '@amplitude/analytics-react-native';
+
+    const eventProperties = {
+        buttonColor: 'primary',
+    };
+    track('Button Clicked', eventProperties);
+    ```
+
+=== "Flutter"
+
+    ```dart title="Dart"
+
+    amplitude.logEvent('BUTTON_CLICKED', {"Hover Time": "100ms"});
+    ```
+
+=== "Unity"
+
+    ```c# title="C#"
+    import 'package:amplitude_flutter/amplitude.dart';
+
+    amplitude.logEvent('MyApp startup', eventProperties: {
+      'friend_num': 10,
+      'is_heavy_user': true
+    });
+    ```
+
+=== "Unreal"
+
+    ```c++ title="C++"
+
+    TArray<FAnalyticsEventAttribute> AppendedAttributes;
+    AppendedAttributes.Emplace(TEXT("Test Event Prop key1"), TEXT("Test Event value1"));
+    AppendedAttributes.Emplace(TEXT("Test Event Prop key2"), TEXT("Test Event value2"));
+    FAnalytics::Get().GetDefaultConfiguredProvider()->RecordEvent(TEXT("Game Started"), AppendedAttributes);
+    ```
+
+=== "GO"
+
+    ```go title="GO"
+
+    amplitude.Track(amplitude.Event{
+           EventType: "Button Clicked",
+           EventOptions: amplitude.EventOptions{
+           UserID:   "user-id",
+           DeviceID: "device-id",
+        },
+        EventProperties: map[string]interface{}{"source": "notification"},
+    })
+
+    ```
+
+## 3. Check for success
+#### Ingestion Debugger
+    1. Login to you Amplitude Analytics Orgnization.
```

Comment:
Probably need to add a new line here, see:
<img width="729" alt="image" src="https://user-images.githubusercontent.com/8689754/188221174-83cd667f-53b4-4fe8-aeff-4d2d4967b4e9.png">